MdePkg/Test/Mock/Include/GoogleTest/Ppi/MockReadOnlyVariable2.h

```diff
@@ -0,0 +1,49 @@
+/** @file MockReadOnlyVariable2.h
+  This file declares a mock of Read-only Variable Service2 PPI.
+
+  This PPI permits read-only access to the UEFI variable store during the PEI phase.
+
+  Copyright (c) Microsoft Corporation.
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+**/
+
+#ifndef MOCK_PEI_READ_ONLY_VARIABLE2_PPI_H

```

Comment:
nit: needs another underscore after "H" for edk2... I can also just add that in the upstream